### ISSUE TITLE
Name for this project is too short

### ISSUE THREAD

**Issue by nicovanzyl** [surfer190/LMNOPQR-LMNOPQR-LMNOPQR-LMNOPQR-LMNOPQR-LMNOPQR-LMNOPQR-LMNOPQR-LMNOPQR-LMNOPQR-LMNOPQR-LMNOPQR-LMNO#1]
The name is way too short, I think you should consider making it a bit more specific by using something longer.

**Comment by surfer190** [comment]
Agreed. It looks like its limited to 100 chars by something. For a compromise, take a look at the description and readme for the specifics.
